style.css

```diff
@@ -313,26 +373,332 @@ h4{
   align-items: center;
 }
 
+.background-section {
+  background: #fff;
+}
+
 .contact-section {
-  background: #6070FF;
-  background-image: url(src/img/contact\ form\ shapes-mobile@2x.png);
+  background: #6070ff;
+  background-image: url(src/img/contact\ form\ shapes-mobile@2x.svg);
   background-repeat: no-repeat;
   background-size: 55%;
   background-position-x: right;
-  background-position-y: -25%;
+  background-position-y: 2.5em;
   border-top-left-radius: 4rem;
 }
 
 .contact-text-section {
   display: flex;
   grid-column: 1/9;
   flex-direction: column;
-  border: 1px solid #dfe1e6;
-  border-top-right-radius: 4em;
+  align-items: center;
 }
 
-.contact-title h4{
-  margin: 2.5em 0.6em 0 0.6em;
+.contact-title h4 {
+  margin: 3em 0.6em 0 0.6em;
   font-size: 2.5em;
   color: #fff;
-}
\ No newline at end of file
+}
+
+form {
+  display: flex;
+  grid-column: 1/9;
+  flex-direction: column;
+  align-items: left;
+  margin: 0.6em 3em 0 0.6em;
+}
+
+.input-box input {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 1em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+.input-box textarea {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  margin: 0.6em 0.6em 0 0.6em;
+  text-align: left;
+  padding: 3%;
+}
+
+button {
+  background-color: #fff;
+  padding: 3%;
+  font-weight: 700;
+  font-size: 1.06em;
+  color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5rem;
+  cursor: pointer;
+  margin: 1em 0.6em 2.5em 0.6em;
+}
+
+@media (min-width: 768px) {
+  body {
+    font-size: 20px;
+  }
+
+  .logo {
+    margin: 0 2% 0 70%;
+  }
+
+  .nav-right img {
+    display: none;
+  }
+
+  .flex-div ul {
+    display: flex;
+    justify-content: space-between;
+    margin: 1em 0 0.8em 60%;
+  }
+
+  .flex-div li {
+    margin-right: 1em;
+    cursor: pointer;
+    color: #344563;
+    font-weight: 550;
+    font-size: 0.7em;
+  }
+
+  .main-container {
+    background-image: url(src/img/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-y: 2em;
+  }
+
+  .greeting-text h1 {
+    margin: 20% 0.6em 0 15%;
+  }
+
+  .greeting-text p {
+    margin: 0.8em 20% 0 15%;
+  }
+
+  .connect p {
+    margin: 0.8em 20% 0 15%;
+  }
+
+  .media-icon {
+    margin: 0.8em 20% 20% 15%;
+    width: 23%;
+  }
+
+  #container-card {
+    align-content: center;
+  }
+
+  .thumbnail .item-1 {
+    position: absolute;
+    left: 5%;
+    width: 42%;
+  }
+
+  .card1 {
+    width: 50%;
+    margin-left: 50%;
+  }
```

Comment:
[OPTIONAL]
- [ ] Please consider making your project card responsive on all screens size. The image of you projects card are overlap on a large screen size, kindly fix it.
### Your design :point_down: 
![Screenshot from 2023-06-10 13-51-49](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/91142432/15ddbaa8-697d-4d5e-a4f7-7f6643f673f6)
### what it is supposed to look like :point_down: 
![Screenshot from 2023-06-10 14-21-11](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/91142432/f89042f8-44b0-42b9-8369-e6366bfd2073)